### ISSUE TITLE
Center of controllers and VR view doesn't align

### ISSUE THREAD

**Issue by ozgurozkan123** [r57zone/Half-Life-Alyx-novr#66]
Room reset inside alyx game when you select controller settings change the alignment but I couldn't find a way to align them easily. I should literally change keyboard mouse direction in real world to make this playable.
https://youtube.com/shorts/GGGV0vwsmg8?feature=share

**Comment by r57zone** [comment]
Hi, Is room reset a feature in Half Life Alyx? What are you using it for?

You need to calibrate in SteamVR as in the instructions, then everything will be fine.

**Comment by ozgurozkan123** [comment]
Yes there is room setup but it ends up center of circles don't align. In game under preferences or the other option there is set controller binding button it brings up htc vive panel GUI on bottom there is a reset room standing button. It changes the center of the controller but it doesn't help me align both centers on the same place or at least on my direction. my left hand or right hand always more closer to me

**Comment by r57zone** [comment]
No idea what could be
